GANA-Core.package/GanaSquitVersionWrapper.class/instance/highlightColor.st

```diff
@@ -0,0 +1,6 @@
+as yet unclassified
+highlightColor
+
+	self historianNames = self defaultHistorian 
+		ifTrue: [^ Color orange]
```

Comment:
collective code ownership - you should have fixed that yourself - now I'll do it <3